### PR TITLE
bump versions

### PR DESCRIPTION
**Pull request by matthyx**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,49 +3,40 @@ module github.com/kubescape/kollector
...
-	go.opentelemetry.io/otel/trace v1.11.2
-	golang.org/x/net v0.5.0
-	k8s.io/api v0.24.3
-	k8s.io/apiextensions-apiserver v0.24.2
-	k8s.io/apimachinery v0.24.3
-	k8s.io/client-go v0.24.3
+	go.opentelemetry.io/otel v1.14.0
+	go.opentelemetry.io/otel/trace v1.14.0
+	golang.org/x/net v0.7.0
+	k8s.io/api v0.26.3
```
Comment (dwertent) [inline go.mod:15]: We should first test it on a GKE cluster before we bump to 1.26

**Review by dwertent — CHANGES_REQUESTED** [review]
You can either revert the k8s packags version updates or build and test on GKE.

File: go.mod (hunk 1 above)
Comment (matthyx) [reply go.mod:15]: We'll test on your cluster after Passover. At least it works on my 1.25

Comment (matthyx) [reply go.mod:15]: @dwertent kollector (quay.io/matthiasb_1/kollector) running on your GKE cluster
(seems to work)

**Review by dwertent — APPROVED** [review]

**Pull request merged**